.github/scripts/requirements.txt

```diff
@@ -1,13 +1,10 @@
-setuptools <65.7.0 ; python_version == '2.7'
-setuptools <=69.1.1 ; python_version >= '3.8'
-pip <23.0 ; python_version == '2.7'
-pip ; python_version >= '3.5'
+setuptools <=69.1.1
+pip
 pylint <2.15.10
 pytest <=8.1.1
 pytest-pylint <=1.1.2
 pytest-runner <7.0.0
 termcolor <2.5.0
 hypothesis <6.100.0
-python-Levenshtein <0.20.9 ; python_version == '2.7'
-levenshtein <=0.25.0 ; python_version >= '3.5'
+levenshtein <=0.25.0
 mock <6.0.0
```

Comment:
This dependency can be removed and replaced with stdlib's unittest.mock.

Reply:
I'm on the case https://github.com/google/python-fire/pull/545